ivy/functional/backends/paddle/layers.py

```diff
@@ -104,7 +104,19 @@ def conv1d_transpose(
     dilations: Union[int, Tuple[int]] = 1,
     out: Optional[paddle.Tensor] = None,
 ):
-    raise IvyNotImplementedException()
+    if data_format == "NCW":
+        x = tf.transpose(x, (0, 2, 1))
+        filters = tf.transpose(filters, (0, 2, 1))
+        output_shape = _output_shape(
+        x.shape, filters.shape, output_shape, strides, padding, 1, dilations
+    )
+    res = tf.nn.conv1d_transpose(
+        x, filters, output_shape, strides, padding, "NWC", dilations
+    )
+    if data_format == "NCW":
+        res = tf.transpose(res, (0, 2, 1))
+    return res
+    #raise IvyNotImplementedException()
```

Comment:
The code doesn't have error handling for when `data_format` is not "NCW" or "NWC". You might want to include some form of validation to ensure the data format is as expected. If it's neither "NCW" nor "NWC", you should raise an error or a warning.

Reply:
Thanks alot for your reply, will make the changes right away! 😊